### ISSUE TITLE
0.9.3 Connection to daemon failed - existing connnection forcibly closed by remote host

### ISSUE THREAD

**Issue by seiferflo** [syncthing/syncthing-gtk#436]
hi. 
0.9.3 is giving me the following error 
![2018-03-13 19-04-37_](https://user-images.githubusercontent.com/10022442/37361421-26026212-26f3-11e8-8e97-d5136bce5e41.gif)
Downgrading to 0.9.2.7 solves the problem.

Syncthing 0.14.45 on W10x64 1803 b17115.1

Let me know if you need any logs. Regards

**Comment by scar45** [comment]
I am experiencing the same thing on Linux, with syncthing daemon version v0.14.45. I don't get a popup message, but the Syncthing-GTK window remains blank, and when I check the About dialog, I see:

>0.9.3 (Daemon unknown)

Starting `syncthing-gtk` from the terminal results in the following output:

```
$>  syncthing-gtk 
I StatusIcon    Using backend StatusIconGTK3 (primary)
I App           
I App           Syncthing-GTK started and running in notification area
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/syncthing_gtk/daemon.py", line 398, in _request_config
    RESTRequest(self, "system/config", self._syncthing_cb_config, self._syncthing_cb_config_error).start()
  File "/usr/lib/python2.7/site-packages/syncthing_gtk/daemon.py", line 1045, in __init__
    self.connect("event", self._socket_event)
TypeError: argument connectable: Expected Gio.SocketConnectable, but got unicode
```

I can try to provide any more information if necessary. Cheers.

**Comment by kozec** [comment]
I believe this is already fixed, I just didn't think it can actually prevent ST-GTK from connecting and didn't released it.

Sorry, I'm little busy these days, but I'll check if that's really the case this evening.

**Comment by scar45** [comment]
Thanks for the update, and no rush at all (at least, in my case). Cheers!

**Comment by kozec** [comment]
Hopefully fixed in [latest release](https://github.com/syncthing/syncthing-gtk/releases/latest)

**Comment by scar45** [comment]
Sorry for the delay, and for necro-commenting, but I just wanted to confirm that this issue is indeed fixed in 0.9.3.1. Cheers!
